### PR TITLE
QE: Add f5 exercise testing in zuul

### PR DESCRIPTION
**Pull request by Spredzy**

### CODE REVIEW

**Comment by Spredzy** [comment]
recheck

**Comment by Spredzy** [comment]
recheck

**Comment by IPvSean** [comment]
recheck

**Comment by IPvSean** [comment]
recheck

**Comment by IPvSean** [comment]
Recheck

**Comment by IPvSean** [comment]
recheck

**Comment by IPvSean** [comment]
@Spredzy I can't get this one to pass... it works fine on my laptop.... not sure what you think

**Comment by Spredzy** [comment]
recheck

File: bindep.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+sshpass [test]
```
Comment (pabelanger) [inline bindep.txt]: change this to
```
sshpass [epel]
```
then we'll add a 2nd call to bindep, once epel is enabled

**Comment by Spredzy** [comment]
recheck

**Pull request merged**